apps/bsatool/bsatool.cpp

```diff
@@ -256,8 +256,8 @@ int extractAll(Bsa::BSAFile& bsa, Arguments& info)
     Bsa::BSAFile::FileList list = bsa.getList();
 
     // Iter on the list
-    for(Bsa::BSAFile::FileList::iterator it = list.begin(); it != list.end(); ++it) {
-        const char* archivePath = it->name;
+    for(auto & it : list) {
```

Comment:
Typically ampersands follow the type without an additional space in the code base.

Reply:
Also Allman?